template/data/achievementinfo.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+
+<dom-module id="achievement-info">
+  <template>
+    <iron-ajax
+      id="ajax"
+      auto
+      url="achievementlist.json"
+      handle-as="json"
+      last-response="{{achievementinfo}}"
+      on-response="filter">
+    </iron-ajax>
+  </template>
+  <script>
+    Polymer({
+      is: 'achievement-info',
+
+      filter: function(e) {
+        this.achievementinfo = e.detail.response[this.achievementid]
+      },
+
+      _updateAchievementInfo: function() {
+        this.$.ajax.generateRequest()
```

Comment:
Beware: This looks ok for now, but the ajax call is not debounced here (no debounce attribute on iron-ajax), so if achievementid changes often and rapidly, this might trigger a lot of ajax calls (and responses might even come back in a different order than the queries where made, since it is asynchronous...).